go-rest-example/auth/auth.go

```diff
@@ -0,0 +1,6 @@
+package auth
+
+// FIXME lol
```

Comment:
nit for this project: Can we make comments as complete sentences? 
eg.
> FIXME: This is not very secure.

Rational:
> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared. 
https://golang.org/doc/effective_go

Reply:
Yes that's fine. This file doesn't exist anymore and was part of the POC I did for rest API, so it wasn't meant for merging or anything